python/lsst/ts/observatory/control/data/mtcalsys.yaml

```diff
@@ -87,3 +87,20 @@ laser_functional:
   use_fiberspectrograph_blue: false
   exposure_times:
     - 15.0
+
+cbp_calibration:
+  calib_type: CBPCalibration
```

Comment:
this is defined as an enum in `ts_observatorycontrol/utils/enums.py`. `calib_type` identifies the light source. So let's keep this as `Mono` and introduce a new variable called `use_cbp` or something like that.